models.js

```diff
@@ -17,3 +17,19 @@ exports.findReviewByID = (id) => {
         : res.rows;
     });
 };
+
+exports.findCommentsByRevID = (id) => {
+  return connection
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.review_id FROM reviews 
+  JOIN comments ON comments.review_id = reviews.review_id
+  WHERE reviews.review_id = $1
+  ORDER BY comments.created_at;`,
+      [parseInt(id.review_id)]
+    )
+    .then((res) => {
```

Comment:
Think about how we can check whether the reason for no data back is due to the article ID not existing or whether it is due to there just not being any comments yet. What do we already have that exists, that would allow us to check this?